### ISSUE TITLE
Issue with s/Num in :query-params 

### ISSUE THREAD

**Issue by earayo** [metosin/compojure-api#317]
Hi Guys.
I'm using `compojure "1.5.1"` and I have the following code on one of my resources:

```clojure
(context "/search" []
   :tags ["search"]

   (GET "/location" []
         :summary "Search users according his location"
         :query-params [{address_lat :- s/Num nil}
                        {address_lon :- s/Num nil}]
         :return {:users [s/Any]}
         (ok {:users [{:id 1 :address_lat address_lat :address_lon address_lon}]})))
```
When I try doing one curl, I always get the following error:
```bash
$ http://localhost:3000/api/search/location?address_lat=39.9546748&address_lon=-75.1944757
{
  "errors": {
    "address_lat": "(not (instance? java.lang.Number '39.9546748'))",
    "address_lon": "(not (instance? java.lang.Number '-75.1944757'))"
  }
}
```
I think it is not be able to do `string->number` coercion in `:query-params`, but I'm not sure.
Who can help me?

Thanks, and Sorry for my english ;)

**Comment by ikitommi** [comment]
Hi. I would also presume that the `string->number` would do this. Would you like to fix this (in ring-swagger)?

**Comment by earayo** [comment]
I'm gonna try :D Thanks!

**Comment by vharmain** [comment]
Meanwhile you can use Double instead of s/Num when coercing coordinates.

**Comment by earayo** [comment]
That's true. Currently we was doing it. Thanks for your suggestion.

**Comment by ikitommi** [comment]
this is shipped both in `[metosin/compojure-api "1.1.11"]` & `[metosin/compojure-api "2.0.0-alpha6"]`. Thanks again!
